ui/packages/shared/components/src/Input/index.tsx

```diff
@@ -74,7 +74,7 @@ const Input = ({
       {hasAction ? (
         <div
           className={cx(
-            '!absolute inset-y-[1px] right-[1px] rounded-bl-none rounded-tl-none rounded-md w-8 z-10'
+            '!absolute inset-y-[1px] right-[1px] rounded-bl-none rounded-tl-none rounded-md w-8 z-4'
```

Comment:
This fixes a visual bug where the filter by function action button is superimposed on top of the metrics graph tooltip.

<img width="657" alt="image" src="https://github.com/parca-dev/parca/assets/9016992/8c412330-af50-48d0-afe5-43227211dce4">